system/objects.rst

```diff
@@ -1,23 +1,23 @@
-Objects
-*******
+Object Attributes
```

Comment:
I don't think we should not change the title of the section, since it's aligned with the UI at the moment. While it's fine to introduce object attributes later in the textx, the section title + link in the sidebar should stay on `Objects`. What do you think?

Reply:
Yes I agree. However, we should consider renaming it in the Zammad UI in the future.